pkg/bucketaws/setup.go

```diff
@@ -49,40 +48,13 @@ func ProcessDate(fromDate time.Time) []string {
 	return dates
 }
 
-// Setting up file formatting
-// Using Viper
-// Reading from file syncron.yaml
-func ConfigRead() {
-	userDirConfig, err := os.UserConfigDir()
-
-	if err != nil {
-		logrus.Fatal(err)
-	}
-
-	viper.SetConfigName("syncron")
-	viper.SetConfigType("yaml")
-	viper.AddConfigPath(userDirConfig)
-	viper.AddConfigPath(".")
-	viper.SetDefault("download_dir", "/tmp/syncron/")
-
-	// Reading from file
-	err = viper.ReadInConfig()
-
-	if err != nil {
-		logrus.Fatal(err)
-	}
-
-	logrus.Info("Your configuration file was read succesfully")
-	logrus.Info("Reading from bucket: ", viper.Get("bucket"))
-}
-
 // Initialize a session with AWS SDK
 // It will read from the file located at ~/.aws/credentials and syncron/syncron.yaml
-func SetupSession() *session.Session {
+func SetupSession(c configuration.Configuration) *session.Session {
```

Comment:
I am nitpicking a lot here, but maybe having a mire self explanatory variable name instead of just "c" would make this a little better.

Reply:
For this `func` it wouldn't be a problem. It's short so it doesn't make the readability worse. But I changed it here and also in one longest `DownloadFromBucket`.